File: livingrimoire start here/LivinGrimoire VB.net/DLC/Personality.vb
Module Personality
    Public Class Personality
        Public Shared Sub Load(brain As Brain)
            brain.addSkill(New DiHelloWorld())
            brain.addSkill(New DiTime())
            brain.addSkill(New DiPrinter())
            ' brain.Chained(New DiHelloWorld()).Chained(New DiTime()).Chained(New DiSysOut())
        End Sub
    End Class


End Module

File: livingrimoire start here/LivinGrimoire C_sharp/DLC/Personality.cs
public class Personality
{
    public static void LoadSkills(Brain brain)
    {
        brain.AddSkill(new DiHelloWorld());
        brain.AddSkill(new DiTime());
        brain.AddSkill(new DiSysOut());
        // brain.Chained(new DiHelloWorld()).Chained(new DiTime()).Chained(new DiSysOut());
    }
}